src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Tools.ServiceModel.Svcutil
 {
     internal class TargetFrameworkHelper
     {
-        internal static readonly List<string> s_currentSupportedVersions = new List<string>() { "8.0", "9.0" };
+        internal static readonly List<string> s_currentSupportedVersions = new List<string>() { "8.0", "9.0", "10.0" };
```

Comment:
Notes:
This list will determine the version of wcf packages  added to the user project. If the detected target framework equals any item in the list, the latest WCF package version (8.*) will be added, otherwise, add WCF 4.10.* packages.

The change here might require a separate PR, but is added here because the current wcf.git code base uses 10.0 SDK for test compilation and execution which leads to unexpected wcf references (4.10) added and causing a bunch of unexpected baseline mismatch test failures.